lib/engine/step/base.rb

```diff
@@ -36,6 +36,10 @@ def did_sell?(_corporation, _entity)
         false
       end
 
+      def may_reduce?(_company)
```

Comment:
don't put this here, since no game uses it cept 48, just have it called in the view the action exists

Reply:
69c49307